File: FreeCodeCamp/Python/spinalCase.py
# https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/spinal-tap-case
# Reprak11

def spinalCase(str):
    str = str.lower().split(" ")
    return "-".join(str)

if __name__ == "__main__":
    print(spinalCase('This Is Spinal Tap'));

File: FreeCodeCamp/Javascript/spinalCase.js
function spinalCase(str) {
    return str.split(/\s|_|(?=[A-Z])/).join("-").toLowerCase();
}

console.log(spinalCase('This Is Spinal Tap'));
console.log(spinalCase("The_Andy_Griffith_Show"));
console.log(spinalCase("thisIsSpinalTap"));